packages/api/src/models/TasksRate.ts

```diff
@@ -0,0 +1,46 @@
+// COULD WE NOT JUST USE MONEY FOR THIS? The only issue with Money is that the amount is a number
```

Comment:
I think we can use Money for it. They both come back as strings in the API.
Eg. tasks:
```
"rate": {
   "amount": "0.00",
   "code": "CAD"
},
```
vs expense
```
"amount": {
   "amount": "54.00",
   "code": "CAD"
},
```